src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int type = (int) (Math.random() * 5);
+        Color color = colorSupplier.getRandomColor();
+
+        switch (type) {
+            case 0:
+                int radius = (int) (Math.random() * 100 + 1);
```

Comment:
create `private final Random random` class-level field and use it
```suggestion
                int radius = random.nextInt(MAX_VALUE);
```